### PR TITLE
update readme with a section on projects using this

### LINKED ISSUE #165: Add list of projects using this
Great project!

Do you think it would be good for adoption and demonstrability if you updated the docs with a section that lists all other consumers of this library?

Very similar to the fody version:
https://github.com/Fody/Home/blob/master/pages/addins.md

As a start you could add your other projects:

https://github.com/andrewlock/StronglyTypedId
https://github.com/AArnott/ImmutableObjectGraph
https://github.com/louthy/language-ext
https://github.com/AArnott/pinvoke

### PR DESCRIPTION
**Pull request by chinwobble**
Closes #165

### CODE REVIEW

**Comment by amis92** [comment]
Link #165

**Comment by amis92** [comment]
First, I think it's not the most important part in the readme, so positioning it as the first is weird.

Second, I think we agreed to name these projects "Plugins" - it makes the intent much clearer than "projects using this project".

**Comment by chinwobble** [comment]
Both good points.
Do you think it should be at the end of the table of contents?

**Comment by amis92** [comment]
Probably. I'd actually consider moving this to wiki for easier updating, and maybe add a link to wiki page in README.

**Comment by amis92** [comment]
Reasoning behind wiki: it's not an information associated with a given version/commit.

File: README.md [hunk 1]
```diff
@@ -10,11 +10,20 @@ up to Intellisense as soon as the file is saved to disk.
...
 * [Developing your code generator][]
 * [Packaging up your code generator for others' use][]
 
+## Projects using CodeGeneration.Roslyn
+[Projects using CodeGeneration.Roslyn]: #Projects using CodeGeneration.Roslyn
+
+* [StronglyTypedId](https://github.com/andrewlock/StronglyTypedId) - generate the boilerplate required to use strongly-typed IDs
+* [ImmutableObjectGraph](https://github.com/AArnott/ImmutableObjectGraph) - make writing immutable objects much easier
+* [C# Functional Programming Language Extensions](https://github.com/louthy/language-ext) - make programming in C# much more reliable and change the engineer's inertia flow in the direction of declarative and functional code rather than imperative.
+* [pinvoke](https://github.com/AArnott/pinvoke) - A collection of libraries intended to contain all P/Invoke method signatures for popular operating systems
```
Comment (AArnott) [inline README.md]: PInvoke's is a consumer (and a generator) whereas `ImmutableObjectGraph` is a generator.
I suspect generator-only projects merit their own subsection so folks can discover and use them in their own projects.

File: README.md [hunk 2]
```diff
@@ -10,11 +10,20 @@ up to Intellisense as soon as the file is saved to disk.
...
 ## Table of Contents
 
+* [Projects using CodeGeneration.Roslyn][]
 * [How to write your own code generator][]
 * [Apply code generation][]
 * [Developing your code generator][]
 * [Packaging up your code generator for others' use][]
 
+## Projects using CodeGeneration.Roslyn
+[Projects using CodeGeneration.Roslyn]: #Projects using CodeGeneration.Roslyn
```
Comment (AArnott) [inline README.md]: This line doesn't seem to render as maybe you expected.

File: README.md [hunk 3]
```diff
@@ -10,11 +10,20 @@ up to Intellisense as soon as the file is saved to disk.
 
 ## Table of Contents
 
+* [Projects using CodeGeneration.Roslyn][]
```
Comment (AArnott) [inline README.md]: I think this belongs at the bottom.

**Comment by amis92** [comment]
@AArnott and what do you think about doing this in wiki instead?

**Comment by AArnott** [comment]
I like that idea too. A link to it from the README is fine.

**Comment by AArnott** [comment]
Moved this data to https://github.com/AArnott/CodeGeneration.Roslyn/wiki/Users

**Pull request closed without merging**